doc/dev/crimson/crimson.rst

```diff
@@ -64,6 +65,57 @@ The last causes pools to be created by default with the ``crimson`` flag.
 Crimson pools are restricted to operations supported by Crimson.
 ``Crimson-osd`` won't instantiate PGs from non-Crimson pools.
 
+Configure Crimson with Bluestore
+================================
+
+Deploying Crimson with Bluestore as the back end requires setting
+several configuration values. The required options are
+``crimson_seastar_cpu_cores`` and ``crimson_alien_thread_cpu_cores``.
+
+Bluestore is not a Crimson native `object store backend`_.
+Therefore, we must explictily set the CPU core allocation for each ``crimson-osd``
+and for the BlueStore back end. It's recommended for each set to be mutually exclusive.
+
+These two options, along with ``crimson_alien_op_num_threads``, can't be changed after
+deployment.
+
+For example, for deploying 2 nodes with 8 CPU cores and 2 OSDs each:
```

Comment:
Do we intend for both OSDs on each node to share the same two cores for bluestore?  That seems odd.

Reply:
Seems to me the important part here is to isolate crimson and bluestore threads.

So in this example, if 8 cpus are allocated to 2 Crimson OSDs, 0-5 are allocated to 2 seastar apps and 6-7 to 2 bluestores.

It doesn't seem critical whether the CPU 6 and 7 are dedicated or shared for the 2 bluestores. And sharing seems simpler and is adopted by vstart IIUC.

Reply:
I've left the text as it was, because I have assumed that the line ``crimson alien thread cpu cores: 6-7`` isolates the BlueStore threads from the Crimson threads. If this is wrong, let me know and I'll make it right.